src/utils/ast-helpers.ts

```diff
@@ -341,6 +341,14 @@ export function isImportPathDeclaration(path: ASTPath): boolean {
   return true;
 }
 
+export function isImportSpecifier(path: ASTPath): boolean {
+  if (!hasNodeType(path.parent, 'ImportDefaultSpecifier') && !hasNodeType(path.parent, 'ImportSpecifier')) {
```

Comment:
should it be `!true && true` instead of `!true && !true`?

Reply:
you mean !(true && true)?